.goreleaser.yml

```diff
@@ -11,31 +13,115 @@ brews:
-  ignore:
-  - goos: windows
-    goarch: arm64
+  env:
+  - CGO_ENABLED=0
   main: ./cmd/insights
+  # goreleaser builds a matrix of the GOOS, GOArch, and GOARM listed below,
+  # minus those under `ignore`.
   goarch:
+  - 386
   - amd64
   - arm
   - arm64
-  - 386
-  env:
-  - CGO_ENABLED=0
   goos:
   - linux
   - darwin
   - windows
   goarm:
   - 6
   - 7
+  ignore:
+  - goos: windows
+    goarch: arm64
 changelog:
   sort: asc
   filters:
     exclude:
       - '^docs:'
       - '^test:'
+      - '^Managed by Terraform$'
+dockers:
+  # There are multiple images to match the `--platform` docker build flag with
+  # combinations of `GOOS`, `GOARCH`, and `GOARM`
+  # Also, some images are broken out to avoid pushing them for pre-release git
+  # tags. E.G. VX.Y.Z-rc1 should not docker-push latest.
+- image_templates:
+  - "quay.io/fairwinds/insights-cli:{{ .Tag }}-amd64"
+  use: buildx
+  build_flag_templates:
+  - "--platform=linux/amd64"
+- image_templates:
+  - "quay.io/fairwinds/insights-cli:v{{ .Major }}-amd64"
+  - "quay.io/fairwinds/insights-cli:v{{ .Major }}.{{ .Minor }}-amd64"
+  use: buildx
+  # Do not push images if the git tag indicates prerelease. E.G. 1.0.0-rc1
+  skip_push: auto
+  build_flag_templates:
+  - "--platform=linux/amd64"
+- image_templates:
+  - "quay.io/fairwinds/insights-cli:{{ .Tag }}-arm64v8"
+  use: buildx
+  goarch: arm64
+  goos: linux
+  build_flag_templates:
+  - "--platform=linux/arm64/v8"
+- image_templates:
+  - "quay.io/fairwinds/insights-cli:v{{ .Major }}-arm64v8"
+  - "quay.io/fairwinds/insights-cli:v{{ .Major }}.{{ .Minor }}-arm64v8"
+  use: buildx
+  # Do not push images if the git tag indicates prerelease. E.G. 1.0.0-rc1
+  skip_push: auto
+  goarch: arm64
+  goos: linux
+  build_flag_templates:
+  - "--platform=linux/arm64/v8"
+- image_templates:
+  - "quay.io/fairwinds/insights-cli:{{ .Tag }}-armv7"
+  use: buildx
+  goarch: arm
+  goarm: 7
+  goos: linux
+  build_flag_templates:
+  - "--platform=linux/arm/v7"
+- image_templates:
+  - "quay.io/fairwinds/insights-cli:v{{ .Major }}-armv7"
+  - "quay.io/fairwinds/insights-cli:v{{ .Major }}.{{ .Minor }}-armv7"
+  use: buildx
+  # Do not push images if the git tag indicates prerelease. E.G. 1.0.0-rc1
+  skip_push: auto
+  goarch: arm
+  goarm: 7
+  goos: linux
+  build_flag_templates:
+  - "--platform=linux/arm/v7"
+docker_manifests:
+# Combine images of the same tag into a single Docker manifest.
+# E.G. An insights-cli:v1.2.3 manifest includes images
+# insights-cli:v1.2.3-amd64, ...:v1.2.3-arm64v8, ...:v1.2.3-armv7, Etc.
+- name_template: quay.io/fairwinds/insights-cli:latest
+  # Do not push manifests if the git tag indicates prerelease. E.G. 1.0.0-rc1
+  skip_push: auto
+  image_templates:
+  - "quay.io/fairwinds/insights-cli:{{ .Tag }}-amd64"
+  - "quay.io/fairwinds/insights-cli:{{ .Tag }}-arm64v8"
+  - "quay.io/fairwinds/insights-cli:{{ .Tag }}-armv7"
```

Comment:
should these be `latest`? They seem identical to the next item below

Reply:
The difference is that the `latest` Docker images and manifest are not built/pushed for pre-releases (alpha, beta), but the images/manifest tagged after the git tag are always built/pushed. Definitely let me know if I'm reading your question incorrectly or not answering it?

Reply:
Robert responded [LGTM in a Slack thread](https://fairwindsops.slack.com/archives/CNQBGRGFN/p1648580550170889).